packages/@expo/fingerprint/src/sourcer/Expo.ts

```diff
@@ -145,13 +145,22 @@ export async function getExpoConfigSourcesAsync(
   return results;
 }
 
-function normalizeExpoConfig(config: ExpoConfig, options: NormalizedOptions): ExpoConfig {
+function normalizeExpoConfig(
+  config: ExpoConfig,
+  projectRoot: string,
+  options: NormalizedOptions
+): ExpoConfig {
   // Deep clone by JSON.parse/stringify that assumes the config is serializable.
   const normalizedConfig: ExpoConfig = JSON.parse(JSON.stringify(config));
 
   const { sourceSkips } = options;
   delete normalizedConfig._internal;
 
+  // googleServicesFile may contain absolute paths on EAS with file-based secrets.
+  // Given we include googleServicesFile as external files already, we can remove it from the config.
+  delete normalizedConfig.android?.googleServicesFile;
+  delete normalizedConfig.ios?.googleServicesFile;
```

Comment:
Hey Kudo, if we delete the keys here, then we will lose access to it [here](https://sourcegraph.com/github.com/expo/expo/-/blob/packages/%40expo/fingerprint/src/sourcer/Expo.ts?L116-119), no?

I think we will have to make a copy of the config and delete it before adding it [here](https://sourcegraph.com/github.com/expo/expo/-/blob/packages/%40expo/fingerprint/src/sourcer/Expo.ts?L48).

Reply:
amazing you are so right! thanks for catching the problem.